protopipe/scripts/write_dl1.py

```diff
@@ -40,6 +40,12 @@ def main():
     parser.add_argument(
         "--regressor_dir", type=str, default="./", help="regressors directory"
     )
+
+    parser.add_argument(
+        "--array_direction_altaz", type=tuple, default=None,
+        help="set an array pointing direction. Required for non parallel pointing"
```

Comment:
I wonder if it's not better to just compute the array direction as the barycenter of the telescope pointings? Then it's automatic and works for both mis-pointing and divergent pointing.

Reply:
The desired array direction could be anything, not necessarily the barycenter of the actual telescopes pointings.

Reply:
However, I can add it as an option? 'barycenter' could be passed for this argument.

Reply:
It's probably a good "default" behavior at least, so the user doesn't have to choose it themselves. I think for divergent pointing, the optimal will always be the barycenter, though, right? At least if you want a roughly circular FOV

Reply:
This CL arguments change has to be replicated also to `write_dl2.py`

Reply:
Ah, and could you also update the docs accordingly?
You just need to copy-paste the new `--help` output in these 2 places,

```
protopipe/docs/scripts/data_training.rst
protopipe/docs/scripts/DL2.rst

```